### PR TITLE
saas connector creation docs

### PR DESCRIPTION
**Pull request by conceptualshark**
Closes DOX-238

### Description of Changes

* Import the documentation on creating your own saas connector from scratch.

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [x] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created

### CODE REVIEW

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+
+A [connection](../configure_connectors) links Fides to you your databases or third-party applications, allowing Fides to retrieve and update selected PII categories when fulfilling privacy requests. Connections are made up of _configurations_, which define how to access your third-party services; and _datasets_, which represent the data to retrieve or mask.
+
+## Create a new connection
+
+### Create a dataset 
+
+A Fides [dataset](../dsr_support/datasets) consists of a declaration of fields, with metadata describing how those fields are related. Fides uses these declarations to know which fields to update or retrieve in your third-party service, and navigates between different collections by referencing their relationships.
+
+The following is example Dataset declaration, representing a simple `conversations` table:
```
Comment (mfbrown) [inline pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:13]: Edit to "The following is an example Dataset declaration, representing a simple `conversations` table:"

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+| `name` | A human-readable name for your dataset.
+| `description` | _Optional_. A human-readable description of your dataset.
+| `collections` | A list of addressable collections which contain the data you would like Fides to retrieve or update.
+
+| `collections` | Description |
+| -- | ----- |
+| `name` | The name of the collection in your dataset must correspond to the name used for it in your datastore. Fides will use this name to generate query and update statements.
+| `fields` | A list of addressable fields in the collection. 
+
+| `fields` | Description |
```
Comment (mfbrown) [inline pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:53]: This isn't rendering correctly:
Screen Shot 2023-01-04 at 3 15 24 PM

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+          - name: <field name> #your third field
+... 
+```
+
+### Add `data_category` annotations
+For each of your fields, provide any relevant data_category information.
+```yaml
+... 
+       fields:
+          - name: id
```
Comment (mfbrown) [inline pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:90]: This needs to be updated to `user.unique_id`

Screen Shot 2023-01-04 at 3 19 59 PM

Comment (mfbrown) [reply pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:90]: To be clear it's the data category that needs to be updated

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+              primary_key: True
+...
+```
+
+The above `primary_key` defines the `id` field as the primary key to use in any queries in the address collection. Additional uses of the `fides_meta` tag can be found in the [dataset guide](../dsr_support/datasets).
+
+## Create a SaaS configuration
+A SaaS [configuration schema](./saas_config) represents a collection of HTTP requests. The configuration defines how to interact with your SaaS application, including how to update and delete the data defined in your dataset.
+
+The following is an example SaaS config declaration, which consists of a single HTTP request, and references the fields defined in the above dataset:
```
Comment (mfbrown) [inline pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:114]: I'm not sure this actually references the above dataset example

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+          - name: conversation_id
+            references:
+              - dataset: mock_dataset
+                field: conversations.id
+                direction: from
+        data_path: conversation_messages        
+```
+
+The above configuration defines the following:
+    *  The domain and authentication requirements for an HTTP client - in this case, [Mailchimp](./example_configs/mailchimp)
```
Comment (mfbrown) [inline pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:153]: Following up on my previous comment, this needs to change OR  we need to update the dataset to use a Mailchimp dataset. 

I'd prefer we use an example dataset like Mailchimp which is probably easier for someone to grasp and reference quickly.

**Review by mfbrown — CHANGES_REQUESTED** [review]

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md (hunk 2 above)
Comment (malaethyca) [reply pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:53]: @suchiroll

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md (hunk 3 above)
Comment (malaethyca) [reply pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:90]: @mfbrown Can you please clarify this for Lalitha?

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md (hunk 1 above)
Comment (mfbrown) [reply pages/fides/dsr_quickstart/saas_connectors/create_connectors.md]: This is in Line 13. Just need to add "an"

File: pages/fides/dsr_quickstart/saas_connectors/create_connectors.md (hunk 2 above)
Comment (mfbrown) [reply pages/fides/dsr_quickstart/saas_connectors/create_connectors.md:53]: Looks like we're missing `| -- | ----- |` between lines 53 and 54

**Comment by ThomasLaPiana** [comment]
@mfbrown do you have the bandwidth to get this over the line? Seems like you're pretty familiar with what was going on here

**Comment by galvana** [comment]
@mfbrown this PR can be replaced by https://github.com/ethyca/fidesdocs/pull/86

**Pull request closed without merging**